verifier/verifier.go

```diff
@@ -518,6 +536,72 @@ func verifyAuthenticTimestamp(outcome *notation.VerificationOutcome) *notation.V
 	}
 }
 
+func verifyRevocation(outcome *notation.VerificationOutcome, client *http.Client, logger log.Logger) *notation.ValidationResult {
+	r, err := revocation.New(client)
```

Comment:
Do we need to create a revocation instance each time? Is the instance reusable?

Reply:
The main rationale for this is if the client changes. Since this can accept a different client, a different revocation object would be needed. Changing the client for a revocation.Revocation object would require access to the revocation struct (which cannot be used outside that package), so we cannot reuse one.

Reply:
This is obsolete in new implementation as the instance is passed, resolving